### ISSUE TITLE
[Bug]: `simplejpeg` Incompatibility with `numpy` Version `>= 2.0.0`

### ISSUE THREAD

**Issue by abhiTronix** [abhiTronix/vidgear#408]
### Description

The backend `simplejpeg` library used in our **WebGear and NetGear APIs** primarily for frame compression, is not compatible with the newer versions of `numpy` (>= 2.0.0). The recent changes in `numpy` have introduced modifications that cause issues with the `simplejpeg` Turbo C API. 

### Issue Checklist

- [X] I have searched open or closed [issues](https://github.com/abhiTronix/vidgear/issues) for my problem and found nothing related or helpful.
- [X] I have read the [Documentation](https://abhitronix.github.io/vidgear/latest) and found nothing related to my problem.
- [X] I've read the [Issue Guidelines](https://abhitronix.github.io/vidgear/latest/contribution/issue/#submitting-an-issue-guidelines) and wholeheartedly agree.

### Expected behaviour

`import simplejpeg` should work out of the box

### Actual behaviour

 Currently, importing `simplejpeg` throws the following error:

```sh
>>> import simplejpeg
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\Abhishek Thakur\AppData\Roaming\Python\Python310\site-packages\simplejpeg\__init__.py", line 1, in <module>
    from ._jpeg import encode_jpeg
  File "simplejpeg\\_jpeg.pyx", line 1, in init simplejpeg._jpeg
ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
```

### Steps to reproduce

- Install `numpy>=2.0.0` and `simplejpeg`
- Call `import simplejpeg`

### Terminal log output

```shell
>>> import simplejpeg
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\Abhishek Thakur\AppData\Roaming\Python\Python310\site-packages\simplejpeg\__init__.py", line 1, in <module>
    from ._jpeg import encode_jpeg
  File "simplejpeg\\_jpeg.pyx", line 1, in init simplejpeg._jpeg
ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
```

### VidGear Version

0.3.3

### Python version

any

### OpenCV version

latest

### Operating System version

any

### Any other Relevant Information?

https://github.com/jfolz/simplejpeg/issues/2

**Comment by abhiTronix** [comment]
### Workaround

As a workaround, downgrading `numpy` to a version `<2.0.0` works for now. If you want to use `simplejpeg`, you can downgrade `numpy` before installing `vidgear`:

```sh
# Downgrade numpy to <2.0.0
pip install "numpy<2.0.0"

# Install vidgear core
pip install vidgear[core]

# Or install vidgear asyncio
pip install vidgear[asyncio]
```

**Comment by abhiTronix** [comment]
**This issue is fixed with [`simplejpeg` version `1.74`](https://pypi.org/project/simplejpeg/1.7.4/).**
